pkg/networkmanager/dialogs-common.jsx

```diff
@@ -35,6 +35,7 @@ import { IpSettingsDialog } from './ip-settings.jsx';
 import { TeamDialog, getGhostSettings as getTeamGhostSettings } from './team.jsx';
 import { TeamPortDialog } from './teamport.jsx';
 import { VlanDialog, getGhostSettings as getVlanGhostSettings } from './vlan.jsx';
+import { WireGuardDialog, getGhostSettings as getWireGuardGhostSettings } from './wireguard.jsx';
```

Comment:
Renaming is confusing. Just rename the original function?

Reply:
That sounds better. I just wanted to be consistent with other renames like that of bond, bridge, team and vlan. Though the name `GhostSettings` itself is confusing for me (maybe it's an actual term).  I assumed it's more like `DefaultSettings` or `InitialSettings`.